doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -392,6 +392,8 @@ Thread samples
 
   All mentioned features are still available in the master variant.
 
+* Added experimental Thread Over Authenticated TLS implementation.
```

Comment:
```suggestion
* Added experimental support for Thread Over Authenticated TLS.
```
Generally how we word things being experimental.

Reply:
fixed.